src/components/dashboard/FaceVerification/hooks/useFaceTecVerification.js

```diff
@@ -53,15 +52,19 @@ export default (options = null) => {
     // destructuring accessors keeping theirs names the
     // same like in the props for avoid code modifications
     const [onUIReady, onCaptureDone, onRetry, onComplete, onError, getMaxRetries] = accessors
+    const isDeviceEmulated = await isEmulator
 
     // if cypress is running
-    if (isE2ERunning || isMobileNative) {
+    // isMobileNative is temporary check, will be removed once we'll deal with Zoom on native
+    if (isE2ERunning || isDeviceEmulated || isMobileNative) {
+      log.debug('skipping fv ui for non real devices', { isMobileNative, isE2ERunning, isDeviceEmulated })
       try {
         // don't start session, just call enroll with fake data
         // to whitelist user on server
-        await api.performFaceVerification({
-          sessionId: uuid(),
-          enrollmentIdentifier,
+        const sessionId = await api.issueSessionToken()
```

Comment:
@sirpy okay, you've took into account that fv is skipped on the dev
but why do you need the real session id ? it would work then with fake session id too because `if (faceVerificationDisabled)` check passed - goodserver won't send any call to the zoom server.  So it would work with the fake session id too in that case

Reply:
dono it jsut didnt work for me without a real session token

Reply:
Okay let's keep this change then